test/unit/helpers/dom/Form.js

```diff
@@ -65,23 +65,25 @@ export default Dom.extend({
     const $container = this.inputWrap(field).parent();
     // container holds input and error description
 
-    const errorId = $container.attr('aria-describedby');
+    // const errorId = $container.attr('aria-describedby');
 
-    if (!errorId) {
-      throw new Error('Expected "aria-describedby" attribute for the error container on field: ' + field);
-    }
+    // Uncomment after OKTA-382383
+    // if (!errorId) {
```

Comment:
Do you plan to commit without this test ?

Reply:
Lester Choi is looking into that ticket now. I can merge this with the commented out test, or I can wait until Lester has a fix and cherry-pick his fix to his PR. I'm open to either

Reply:
I suspect if any selenium test fails in okta-core when you try to merge..  If Lester is working actively in plan to merge this week, then i guess you can go ahead if bacon is green for okta-core..

You can kick a downstream build in bacon to see if okta-core is green for this widget change..

Reply:
`okta-core` tests have already passed on Bacon, and I've merged the ui and `okta-core` PRs. I also saw all the SIW tests passed on Bacon after cherry-picking Patrick's commit and commenting out this test here.

Reply:
I meant when you merge this change to SIW and then widget gets upgraded to okta-core, then okta-core should pass..

Reply:
Ah, I didn't think about that! Lots of moving parts on this change haha. Ok, I'll make sure that passes before merging this one

Reply:
Looks like it passed: https://bacon-go.aue1e.saasure.net/commits?artifact=monolith&branch=d16t-okta-signin-widget-f7e9d9c-60664087